src/System.Private.ServiceModel/tests/Scenarios/Client/ExpectedExceptions/ExpectedExceptionTests.cs

```diff
@@ -10,47 +12,6 @@ public static class ExpectedExceptionTests
 {
     [Fact]
     [OuterLoop]
-    public static void NotExistentHost_Throws_EndpointNotFoundException()
```

Comment:
Be aware Theory is not supported yet in the experimental NET Native test environment, so it will make the test a NOP in NET Native.  I recommend avoiding Theory for now except for strictly data driven tests.